vignettes/Intro.Rmd

```diff
@@ -73,6 +73,21 @@ log_eval(y <- f(g(x)), level = INFO)
 str(y)
 ```
 
+
+Sometimes, it may be reasonable to want to log a smallish data.frame or data.table, e.g. mtcars.
+
+```{r}
+formatter_data_frame <- function(df, ...) {
```

Comment:
is there any good reason to define a custom formatter here instead of using `formatter_pander`?

Reply:
Good point. How is it used?

I am okay with using `formatter_pander` as long as there is a decent example in the docs showing how it is used. 

```R
library(logger)
logger::log_info(mtcars)
```

Should work "out of the box" or have a ready example in the docs of how to solve the error.

Here is the error:
```R
Error in value[[3L]](cond) : `glue` failed in `formatter_glue` on:

 'data.frame':  32 obs. of  11 variables: 

Raw error message:

 All unnamed arguments must be length 1 

Please consider using another `log_formatter` or `skip_formatter` on strings with curly braces.`glue` failed in `formatter_glue` on:

  $ mpg : num  21 21 22.8 21.4 18.7 18.1 14.3 24.4 22.8 19.2 ... 

Raw error message:

 All unnamed arguments must be length 1 

Please consider using another `log_formatter` or `skip_formatter` on strings with curly braces.`glue` failed in `formatter_glue` on:

  $ cyl : num  6 6 4 6 8 6 8 4 4 6 ... 

Raw error message:

 All unnamed arguments must be length 1 

Please consider using another `log_formatter` or `skip_formatter` on strings with curly braces.`glue` failed in `formatter_glue` on:

  $ disp: num  160 160 108 258 360 ... 

Raw error message:

 All unnamed arguments must be length 1 

Please consider using another `log_formatter` or `skip_formatter` on strings with curly braces.`glue` failed i
```

Reply:
Shouldn't `log_formatter(formatter_pander)` be mentioned in the docs and or the error message when a user tries to log a data frame ?